lib/ansible/plugins/connection/ssh.py

```diff
@@ -1115,6 +1142,18 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
 
         display.vvv(u"ESTABLISH SSH CONNECTION FOR USER: {0}".format(self._play_context.remote_user), host=self._play_context.remote_addr)
 
+        if self._shell.SHELL_FAMILY == 'powershell':
+            # Become method 'runas' is done in the wrapper that is executed,
+            # need to disable sudoable so the bare_run is not waiting for a
+            # prompt that will not occur
+            sudoable = False
+
+            # Make sure our first command is to set the console encoding to
+            # utf-8, this must be done to get unicode output chars instead of ?
+            cmd_parts = ["chcp.com", "65001", self._shell._SHELL_REDIRECT_ALLNULL, self._shell._SHELL_AND]
```

Comment:
Change above comment to explicitly mention that `chcp.com` is to change the encoding and `65001 == utf-8`.